new-implementation/aws/backends/eks/variables.tf

```diff
@@ -62,6 +62,9 @@ variable "cluster_endpoint_public_access_cidrs" {
                   "193.240.221.236/32",  # Theale and Belfast
                   "58.76.201.220/32",    # South Korea
                   "80.75.109.253/32",    # fuzztds office
+		  "35.202.29.179/32",
```

Comment:
thanks @ksripathi for this!  can you 1. make sure to run `tflint` or `terraform fmt on all this` and 2. make sure to sync all ips for all three clouds.

Reply:
done, yup all cloud IPs list is set to same